### PR TITLE
Riato 582 get stats

### PR DESCRIPTION
**Pull request by skyproudmanp**
Summary: Get stats for the number of rendered and dropped frames
Type: Fix  
Test Plan: UT, CT, test that wpe-webkit can get stats on XiOne US
Jira: RIALTO-582

### CODE REVIEW

**Comment by aczs** [comment]
Please make PR title more informative

File: media/server/gstplayer/source/GstGenericPlayer.cpp [hunk 1]
```diff
@@ -382,6 +382,69 @@ bool GstGenericPlayer::getPosition(std::int64_t &position)
...
+        {
+            // For AutoVideoSink we set properties on the child sink
+            sink = getSinkChildIfAutoVideoSink(sink);
+
+            GstStructure *stats{nullptr};
+            m_glibWrapper->gObjectGet(sink, "stats", &stats, nullptr);
+            if (!stats)
+            {
+                RIALTO_SERVER_LOG_WARN("failed to get stats");
+                return false;
```
Comment (aczs) [inline media/server/gstplayer/source/GstGenericPlayer.cpp:422]: Isn't sink leaked here?

**Review by skywojciechowskim — APPROVED** [review]

File: media/server/gstplayer/source/GstGenericPlayer.cpp [hunk 2]
```diff
@@ -382,6 +382,68 @@ bool GstGenericPlayer::getPosition(std::int64_t &position)
...
+        break;
+    case MediaSourceType::VIDEO:
+        kSinkName = "video-sink";
+        break;
+    default:
+        break;
+    }
+    if (!kSinkName)
+    {
+        RIALTO_SERVER_LOG_WARN("mediaSourceType not supported %d", static_cast<int>(mediaSourceType));
```
Comment (aczs) [inline media/server/gstplayer/source/GstGenericPlayer.cpp:402]: use common::convertMediaSourceType for print string name of the media source

File: media/server/gstplayer/source/GstGenericPlayer.cpp [hunk 3]
```diff
@@ -382,6 +382,68 @@ bool GstGenericPlayer::getPosition(std::int64_t &position)
...
+        else
+        {
+            // For AutoVideoSink we set properties on the child sink
+            sink = getSinkChildIfAutoVideoSink(sink);
+
+            GstStructure *stats{nullptr};
+            m_glibWrapper->gObjectGet(sink, "stats", &stats, nullptr);
+            if (!stats)
+            {
+                RIALTO_SERVER_LOG_WARN("failed to get stats");
```
Comment (aczs) [inline media/server/gstplayer/source/GstGenericPlayer.cpp:421]: add info from which sink it failed

**Review by aczs — APPROVED** [review]

**Review by skywojciechowskim — APPROVED** [review]

**Pull request merged**